api/hardwaremanagement/v1alpha1/node.go

```diff
@@ -39,7 +46,7 @@ type BMC struct {
 type NodeStatus struct {
 	BMC *BMC `json:"bmc,omitempty"`
 
-	BootMACAddress string `json:"bootMACAddress,omitempty"`
+	Interfaces []*Interface `json:"interfaces,omitempty"`
```

Comment:
Going forward, we'll need to be careful about changes like this, coordinating changes with the hwmgr plugins (where we currently only have the Test Plugin). For example, we may need to keep stale fields around for a period to avoid breaking the plugin(s). Since the changes to enable the Test Plugin integration are still under review, I'd be the only one impacted, but this would definitely be a breaking change for the Test Plugin, ie. a Non-Upward Compatible change (NUC)

Reply:
Agree.